transactions/repository.go

```diff
@@ -0,0 +1,78 @@
+package transactions
+
+import (
+	"github.com/centrifuge/go-centrifuge/errors"
+	"github.com/centrifuge/go-centrifuge/storage"
+	"github.com/centrifuge/go-centrifuge/utils"
+	"github.com/ethereum/go-ethereum/common"
+	"github.com/satori/go.uuid"
```

Comment:
What is the benefit of using a generic uuid?

Why not something specific with addtitional information.

For example: `concat(identityAddress,sha3(transactionCounter))`

Reply:
I just notice on the db level we storing the uuid anyway together with the identityAddress

Reply:
Storing uuid alone is preferable but if we want tenant specific transactions, its going to be inefficient

Hence, we prefix it with identity so that its easier to fetch tenant specific txns